packages/wonder-blocks-dropdown/components/option-item.js

```diff
@@ -34,15 +33,14 @@ type OptionProps = {|
     /**
      * Optional user-supplied callback when this item is called.
      */
-    onClick?: (oldSelectionState: boolean) => void,
```

Comment:
The user of the menu controls which items are selected, so they already have this information.